3Dmol/glmodel.js

```diff
@@ -2468,8 +2496,10 @@ $3Dmol.GLModel = (function() {
                     console.log(err);
                 }
             }
-            if (format == "mdcrd" || format == "inpcrd" || format == "pdb" || format == "netcdf") {
-                frames = [];
+            var supportedFormats = ["mdcrd","inpcrd","pdb","netcdf","unknown"];
```

Comment:
As a general rule of thumb, don't use lists for membership testing as it is O(n).  If you are going to query a data structure, make them keys in an object.